### PR TITLE
Jm delete payment test

### LINKED ISSUE #17: TEST: Delete payment type
Add a test to the `tests/payment.py` module that verifies that a payment type can be deleted.

### PR DESCRIPTION
**Pull request by medlenmage**
Created a test for deleting payments.

## Requests / Responses

**Request**

DELETE `/paymenttypes/:id` Deletes a payment type

**Response**

HTTP/1.1 200 OK

## Testing

Description of how to test code...

- [ ] Open `tests/payments.py`
- [ ] Run `python manage.py test tests -v 1`   
- [ ] check for failures/errors

## Related Issues

- closes #17

### CODE REVIEW

File: tests/payments.py [hunk 1]
```diff
@@ -40,4 +41,20 @@ def test_create_payment_type(self):
...
+        payment.expiration_date = '2023-12-15'
+        payment.create_date = '2020-12-15'
+        payment.save()
+
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        response = self.client.delete(f"/paymenttypes/{payment.id}")
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        response = self.client.get(f"/paymenttypes/{payment.id}")
+        self.assertEqual(response.status_code, status.HTTP_500_INTERNAL_SERVER_ERROR)
```
Comment (BrianBNeal) [inline tests/payments.py:60]: Does this test pass? It should return a 404 if the thing you're trying to get doesn't exist.

**Review by BrianBNeal — CHANGES_REQUESTED** [review]

File: tests/payments.py (hunk 1 above)
Comment (medlenmage) [reply tests/payments.py:60]: it does pass. the create payment is throwing an error but it was throwing an error before i created the delete payment test. i was gonna address it in a separate ticket.

Comment (medlenmage) [reply tests/payments.py:60]: image


that's the only error but its for create payment. unless i'm over looking something.

Comment (BrianBNeal) [reply tests/payments.py:60]: Just realized the 404 is returned from the destroy and the 500 you're asserting is from the retrieve, so nevermind!

**Review by BrianBNeal — APPROVED** [review]

**Pull request merged**